orangecontrib/text/vectorization/base.py

```diff
@@ -28,3 +29,6 @@ def add_features(corpus, X, dictionary):
         corpus.extend_attributes(X[:, order],
                                  feature_names=(dictionary[i] for i in order),
                                  var_attrs={'hidden': True})
+        corpus.ngrams_dictionary = Dictionary()
```

Comment:
Is here `ngrams_dictionary` set to the same thing as `dictionary` if `ngram_range = (1, 1)`? Such cases are the reason I would prefer just one dictionary.